src/user.spec.js

```diff
@@ -0,0 +1,100 @@
+/* eslint-env jest */
+
+import {
+    getUser,  
+    testConnection,
+    xo,
+  } from "./util";
+  
+  describe("user", () => {
+    let userId1, userId2;
```

Comment:
Declare a `Set` to stoke user ids and delete them in a `afterEach` block. Like the old implementation.

Reply:
I use the two same users in all my tests, created at the beginning and will be deleted at the end.
I can put them into a set but if I delete them after each block, should I create other users for each block ?